README.md

```diff
@@ -38,7 +38,17 @@ To get started:
 1. At this point, you should be able to run the notebooks in this repo. Explore our notebooks on the following computer vision domains. Make sure to change the kernel to "Python (cv)".
     - [/classification](classification#notebooks)
     - [/similarity](similarity#notebooks)
-    
+
+
+The utilities in [`utils_cv`](utils_cv) is `pip`-installable from the
```

Comment:
Bit unclear. Maybe use this text here instead: "As an alternative to the steps above, and if one wants to instal only the 'utils_cv' library (without creating a new conda environment), this can be done by running 
```bash 
pip install git+https://github.com/microsoft/ComputerVision.git@master#egg=utils_cv 
```
or by downloading the repo and then running `pip install .` in the root directory.

Reply:
Changed.